DynamicData.Profile/DynamicData.Profile.csproj

```diff
@@ -10,6 +10,7 @@
   
   <ItemGroup>
     <PackageReference Include="System.Reactive" Version="4.0.0" />
+    <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" />
```

Comment:
Wouldn't this be picked up due to the fact it's in the Directory.Build.Props file earlier, so this is redundant?

Reply:
The other issue is you're not doing PrivateAssets="all" so it will cause nuget users of this framework to have to inherit this nuget package.